digital_image_processing/feature_detectors/README.md

```diff
@@ -0,0 +1,20 @@
+# Computation of Harris Detector
```

Comment:
I would be really helpful if you can add this file in [Algorithms-Explanation](https://github.com/TheAlgorithms/Algorithms-Explanation) repo. That will help us in future to organise the code and documentation and parse them using scripts.

Reply:
Already Added. Pull request 34. [https://github.com/TheAlgorithms/Algorithms-Explanation/pull/34](url).